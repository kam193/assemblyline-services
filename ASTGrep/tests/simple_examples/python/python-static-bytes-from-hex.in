bytes.fromhex("696d706f7274206f73")
bytes.fromhex(this_is_a_test)
