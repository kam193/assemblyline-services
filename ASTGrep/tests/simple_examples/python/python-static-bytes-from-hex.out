b'import os'
bytes.fromhex(this_is_a_test)
